components/persistor/package.json

```diff
@@ -21,23 +21,23 @@
     },
     "devDependencies": {
         "@havenlife/supertype": "3.x",
-        "chai": "3.x",
-        "chai-as-promised": "^5.1.0",
-        "nyc": "13.0.0",
-        "mocha": "5.2.0",
-        "sinon": "1.15.x",
-        "sinon-chai": "2.8.x",
-        "mock-fs": "3.6.x",
-        "eslint": "3.7.x",
-        "reflect-metadata": "^0.1.13",
+        "@types/bluebird": "*",
```

Comment:
The package.json / package-lock.json seems to be really out of date.  Let me know if I should update the package.json/package-lock.json with an older npm version.